text/0000-chmod.md

```diff
@@ -0,0 +1,62 @@
+# Chmod Buildpack
+
+## Summary
+
+A chmod buildpack that allows users to change directories and files (produced at buildtime) UNIX permissions.
+
+This buildpack would be configured to run last, so that it would be able to change any directories and files produced by previous buildpacks.
+
+## Motivation
+
+Buildstacks and Builders rely on Stacks.
+
+Stacks define build time and runtime base images.
+
+Some stacks do not use the same user / UID between build and run images, [for security reasons](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0045-user-ids.md), since a runtime user should just be able to run a process; not create directories nor install packages.
+
+Unfortunately, some buildpacks do need to create directories and files: they run setup scripts during runtime that fail because they can't write folders nor files, since they were run with a runtime user, who can be restricted with some Stacks. 
+
+Similarly, some scripts that need to be run at runtime are only executable by the build time user, and would need to be executable by the runtime user.
+
+## Detailed Explanation
+
+N/A
+
+## Rationale and Alternatives
+
+We have considered solving the issue at different levels:
+
+* At the stack level: reverting to former stacks that define the same user at build time and runtime.
+
+After all, previous stacks like Ubuntu Bionic did not differentiate the runtime user from the build time user, both being named `cnb` with the UID 1000
+
+We began noticing this issue with the latter stacks (some based on Ubuntu Jammy) that guarantee [a more secure environment](https://github.com/buildpacks/rfcs/blob/main/text/0085-run-uid.md) by having the build time user set to UID 1001, and runtime user set to 1000 at runtime.
```

Comment:
The Paketo stacks actually have a run-time UID of `1002`. Other commercial stacks may have run-time UIDs of `1000` or other values.

Regardless, your general point still stands.

Reply:
```suggestion
We began noticing this issue with the latter stacks (some based on Ubuntu Jammy) that guarantee [a more secure environment](https://github.com/buildpacks/rfcs/blob/main/text/0085-run-uid.md) by having the build time user set to UID 1002, and runtime user set to 1000 at runtime.
```